test/src/main/java/org/apache/accumulo/test/functional/MetadataIT.java

```diff
@@ -156,4 +171,122 @@ public void batchScanTest() throws Exception {
       }
     }
   }
+
+  @Test
+  public void testAmpleReadTablets() throws Exception {
+
+    try (AccumuloClient accumuloClient = Accumulo.newClient().from(getClientProps()).build()) {
+      accumuloClient.securityOperations().grantTablePermission(accumuloClient.whoami(),
+          MetadataTable.NAME, TablePermission.WRITE);
+      BatchWriter bw =
+          accumuloClient.createBatchWriter(MetadataTable.NAME, new BatchWriterConfig());
+      ClientContext cc = (ClientContext) accumuloClient;
+      // Create a fake METADATA table with these splits
```

Comment:
Why not just use the API to create a table with those splits and let Accumulo populate the metadata?

Reply:
That idea is more economical. I guess an effective test does not always need a full-blown client. I will update the test.